media-libs/liblscp/liblscp-0.5.8-r1.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="C++ library for the Linux Sampler control protocol"
+HOMEPAGE="https://www.linuxsampler.org"
+SRC_URI="https://download.linuxsampler.org/packages/${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="doc"
+
+DEPEND="doc? ( app-doc/doxygen )"
+
+DOCS=( AUTHORS ChangeLog TODO NEWS README )
+
+src_install() {
+	use doc && local HTML_DOCS=( doc/html/ )
```

Comment:
almost ;) missing trailing `.` means the package installs to `html/html/`

Reply:
Ha.. you're right. Didn't notice the wrong path :)
Thanks.. fixed now.